providers/network/config/config.go

```diff
@@ -30,9 +30,18 @@ var Config = plugin.Provider{
 	},
 	Connectors: []plugin.Connector{
 		{
-			Name:      "host",
-			Use:       "host HOST",
-			Short:     "a remote host",
+			Name:  "host",
+			Use:   "host HOST",
+			Short: "a remote host",
+			Long: `Use the host provider to query remote hosts. 
```

Comment:
We probably want to slip HTTP/HTTPS into this description since "host" is way way generic